What’s up in Vietnam.md

```diff
@@ -1,12 +1,12 @@
 # What’s up in Vietnam
 ## Introduction
-Is there a more sophisticated parallel to a part of computer science than a situation in Vietnam? The first brave soul was Ted Neward with his article “The Vietnam of Computer Science” comparing the Object Relational Mapping to one of the most disgraceful conflicts in the history of United States. It was written in 2006. 12 years have passed, and a lot has changed in the approach to ORM.
+Is there a more sophisticated parallel to a particular part of computer science than the situation in Vietnam? The first brave soul was Ted Neward with his article “The Vietnam of Computer Science” comparing Object Relational Mapping to one of the most disgraceful conflicts in the history of United States. It was written in 2006. 12 years have passed, and a lot has changed in the approach to ORM.
 
 ## What the Object Relational Mapping stands for?
-Follow the definition on Wikipedia _it’s a technique for converting data between incompatible type systems using object-oriented programming languages._ To be more precise it’s converting relational data to the object-oriented domain model (OODM) and OODM to relational data. Conversions like that can be done in a several ways, but the most common approach is to use a tool called ORM library or ORM framework.
+Following the definition on Wikipedia _it’s a technique for converting data between incompatible type systems using object-oriented programming languages._ To be more precise it’s converting relational data to the object-oriented domain model (OODM) and OODM to relational data. Conversions like that can be done in several ways, but the most common approach is to use a tool called ORM library or ORM framework.
 
 ## Why is it problematic?
- „Incompatible type systems” are mentioned already in the definition, and all incompatibilities are potential problems. My first question that popped into my head after getting into the subject was: „Why the hell we are using incompatible systems? Why is it the first (and often the only one known) solution to solve the problem with persisting data?”
+ „Incompatible type systems” are mentioned already in the definition, and all incompatibilities are potential problems. First question that popped into my head after getting into the subject was: „Why the hell are we using incompatible systems? Why is it the first (and often the only) solution to solve the problem with persisting data?”

```

Comment:
`the only` without `the only [one] known` changes context a little bit. The question is: is it bad ... hmmm.

Reply:
Question is what is it that you wanted to say. 'The only one' suggests it is the only one, 'quite often the only one ' makes no sense I think, hence I added 'known' :) Probably just needs to be rephrased to pass carry the meaning you wanted it to carry.